modules/core/include/opencv2/core/matx.hpp

```diff
@@ -714,6 +720,18 @@ Matx<_Tp, m, 1> Matx<_Tp, m, n>::col(int j) const
 }
 
 template<typename _Tp, int m, int n> inline
+void Matx<_Tp, m, n>::copyTo(Matx<_Tp, m, n>& matx) const
```

Comment:
The default copy constructor or the default assignment operator will do this for you.